WordPress/WordPressStatsWidgets/LockScreenWidgets/LockScreenStatsWidget.swift

```diff
@@ -0,0 +1,33 @@
+import WidgetKit
+import SwiftUI
+
+@available(iOS 16.0, *)
+struct LockScreenStatsWidget: Widget {
+    private let placeholderContent = HomeWidgetTodayData(
+        siteID: 0,
+        siteName: "My WordPress Site",
+        url: "",
+        timeZone: TimeZone.current,
+        date: Date(),
+        stats: TodayWidgetStats(views: 649,
+                                visitors: 572,
+                                likes: 16,
+                                comments: 8))
+
+    var body: some WidgetConfiguration {
+        IntentConfiguration(
+            kind: AppConfiguration.Widget.Stats.lockScreenTodayViewsKind,
+            intent: SelectSiteIntent.self,
+            provider: SiteListProvider<HomeWidgetTodayData>(
+                service: StatsWidgetsService(),
+                placeholderContent: placeholderContent,
+                widgetKind: .today
+            )
+        ) { (entry: StatsWidgetEntry) -> LockScreenStatsWidgetsView in
+            return LockScreenStatsWidgetsView(timelineEntry: entry)
+        }
+        .configurationDisplayName(LocalizableStrings.todayWidgetTitle)
+        .description(LocalizableStrings.todayPreviewDescription)
+        .supportedFamilies(FeatureFlag.lockScreenWidget.enabled ? [.accessoryRectangular] : [])
```

Comment:
Out of curiosity, is this `[.accessoryRectangular]` the line that enables the widget to appear on the lock screen?

Reply:
Yes, `accessoryRectangular`, `accessoryInline`, and `accessoryCircular` can appear in watchOS or on the Lock Screen in iOS.

So another solution is we can just extend existing widget support by adding `accessoryRectangular`,
and provide the view based on the currently family, but that's not good for scalable when more widgets in the future :)

Reply:
Nice, thanks for explaining!